rfcs/QueryLevelNullability.md

```diff
@@ -126,16 +188,25 @@ query {
 ```
 
 ## ✅ RFC Goals
+
 - Non-nullable syntax that is based off of syntax that developers will already be familiar with
+- Enable GraphQL codegen tools to generate more ergonomic types
 
 ## 🚫 RFC Non-goals
 
 ## 🗳️ Alternatives considered
 
-### Use a directive such as `@nonNull` instead
-This is the alternative being used at some of the companies represented in this proposal for the time being.
-However this feels like a common enough need to call for a language feature. A single language feature
-rather than using directives as a workaround would enable more unified public tooling around GraphQL.
+### An official `@nonNull` directive
+
+This solution offers the same benefits as the proposed solution. Since many GraphQL codegen tools already support the `@skip` and `@include` directives, this solution likely has a faster turnaround.
+
+### Use a custom `@nonNull` directive
```

Comment:
like we discussed over Slack, I added a new alternative here and tried to make it more clear that the original alternative solution was for custom directives only